scheduler/stack_oss.go

```diff
@@ -0,0 +1,78 @@
+// +build !pro,!ent
+
+package scheduler
+
+// NewGenericStack constructs a stack used for selecting service placements
+func NewGenericStack(batch bool, ctx Context) *GenericStack {
```

Comment:
Do we potentially want to have a wait to modify the stack, rather than duping the whole setup? (mostly worried about duplicate work when introducing other new scheduler things that could easily be forgotten to be also PR'd to ENT)